docs/guide.md

```diff
@@ -304,6 +331,22 @@ for passing into a Docker instance:
 $ export PL_API_KEY=$(planet auth value)
 ```
 
+
+### Collecting Results
+
+Some API calls, such as searching for imagery and listing orders, return a
+varying, and potentially large, number of results. These API responses are
+paged. The SDK manages paging internally and the associated cli commands
+output the results as a sequence. These results can be converted to a JSON blob
+using the `collect` command. When the results
+represent GeoJSON features, the JSON blob is a GeoJSON FeatureCollection.
+Otherwise, the JSON blob is a list of the individual results.
+
+```console
+$ planet orders list | planet collect -
```

Comment:
I'd probably make the main example to be the data search-quick one, since that'll be the most common.

Also why is it with a '-'? Might be worth to explain that somewhere. I think we probably should have a 'CLI primer' somewhere that covers piping results and the like, as I've had to level up a bit to use some of these things. I like it overall, it's quite powerful, but we need to lead people to it.

Reply:
good points. I made a ticket for the cli primer. The `-` for stdin is the format we've been using. Here, and elsewhere, we can have the `-` be the default value so it's not required. I'll make the move to `planet data search-quick`